File: S1/C#Project/Java/CH13/11月23日晚自习作业/cn/jbit/Student.java
package cn.jbit;

public class Student {
	int score;
	String grade;
	public String getGrade(int score){
		if(score >= 85 && score <= 100){
			grade = "A";
		}else if(score >= 70 && score <= 84){
			grade = "B";
		}else if(score >= 60 && score <= 69){
			grade = "C";
		}else{
			grade = "D";
		}
		return grade;
	}
}

File: S1/C#作业/2月18日下午作业/2月18日下午作业/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _2月18日下午作业
{
    public class Student
    {
        public int sno;
        public string sname;
        public int grade;
        public void showInfo()
        {
            Console.WriteLine("学号是：" + sno + "，姓名是：" + sname + "，成绩是：" + grade);
        }

    }

    class TestStudent
    {
        static void Main(string[] args)
        {
            Student student = new Student();
            Console.WriteLine("输入您的学号：");
            student.sno = int.Parse(Console.ReadLine());
            Console.WriteLine("输入您的姓名：");
            student.sname = Console.ReadLine();
            Console.WriteLine("输入您的成绩：");
            student.grade = int.Parse(Console.ReadLine());
            student.showInfo();
            Console.ReadLine();
        }
    }
}
